DuckDuckGo/OmniBar.swift

```diff
@@ -121,13 +121,19 @@ class OmniBar: UIView {
         if textField.isEditing {
             return
         }
-        
+
         guard let url = url else {
             textField.text = nil
             return
         }
-        
-        textField.text = url.absoluteString
+
+        let query = appUrls.searchQuery(fromUrl: url)
+
+        if query != nil {
```

Comment:
The convention here would normally be:

`if let query = appUrls.searchQuery(fromUrl: url) {`